Resources/Prototypes/_Sunrise/Loadouts/loadout_groups.yml

```
@@ -225,6 +225,18 @@
   - SunriseWeaponEnergyGun
   - SunriseWeaponTaser
 
+# Security  Detective
+- type: loadoutGroup
+  id: SunriseWeaponDetective
+  name: loadout-group-security-gun
+  minLimit: 1
+  maxLimit: 1
+  loadouts:
+  - DetectiveWeapon
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)